README.md

```diff
@@ -171,6 +171,12 @@ Run the `destroy-cloudfoundry` pipeline.
 
 # Additional notes
 
+## Running tests locally
+
+You will need to install some dependencies to run the unit tests on your own
+machine. The most up-to-date reference for these is the Travis CI
+configuration in [`.travis.yml`](.travis.yml).
+
```

Comment:
I wonder whether we need a little more detail around installing things from pip. It took me a while to figure out where `pip install --user` had put `yamllint` because it wasn't on my path by default.